qa/c8-orchestration-cluster-e2e-test-suite/pages/UtilitiesPage.ts

```diff
@@ -53,10 +54,31 @@ export async function completeTaskWithRetry(
         taskDetailsPage.detailsPanel.getByText(taskPriority),
       ).toBeVisible();
       await taskDetailsPage.clickCompleteTaskButton();
-      await expect(
-        taskPanelPage.availableTasks.getByText(taskName, {exact: true}).first(),
-      ).not.toBeVisible({timeout: 5000});
-      return;
+      let assertionPassed = false;
```

Comment:
I also think it's not a good idea to have these custom retry mechanisms
This probably could be replaced with `toPass`